# PredictION
A predictive model to establish performance of Oxford sequencing reads of SARS-CoV-2

With PredictION you may estimate sequence reads given a priori known concentration of cDNA (ng/µl) per sample and the desired coverage depth (mean) and coverage per genome (percentage)

# Authors

David E. Valencia-Valencia 1,§; Diana López-Alvarez 1,2,3,§; Nelson Rivera Franco 1,2; Andres Castillo 1; Johan S. Piña 4; and Beatriz Parra 2

1. Laboratorio de Técnicas y Análisis Ómicos - TAOLab/CiBioFi, Facultad de Ciencias Naturales y Exactas, Universidad del Valle, Cali, Colombia
2. Grupo VIREM - Virus Emergentes y Enfermedad, Escuela de Ciencias Básicas, Facultad de Salud, Universidad del Valle, Cali, Colombia
3. Departamento de Ciencias Biológicas, Facultad de Ciencias Agropecuarias, Universidad Nacional de Colombia, Palmira, Colombia
4. Department of Data Science, People Contact, Manizales, Caldas, Colombia
§  Both authors contributed equally to this worK
 
# Acknowledgments
We are thankful to all the VIREM team (“Virus Emergentes y Enfermedades”), who are supporting the diagnostics of SARS-CoV-2 in Colombia, as well as different investigators of Colombia and worldwide who deposited their genomes in GISAID. We also thank the Colombian network for SARS-CoV-2 genomic surveillance led by the Colombian National Institute of Health (INS). Thanks to CEO Diego Ceballos of People Contact enterprise for supporting the dashboard builder. 

# Funding Statement
The study was supported by NIH R01NS110122 to the Neurovirus Emerging in the Americas Study (NEAS).

# Disclaimer
The content of this research code repository (i) is not intended to be a medical device; and (ii) is not intended for clinical use of any kind, including but not limited to diagnosis or prognosis.
